packages/web/src/components/social-button/tiktok-button/TikTokButton.tsx

```diff
@@ -1,18 +1,7 @@
-import { IconTikTok } from '@audius/harmony'
-import cn from 'classnames'
+import { SocialButton, SocialButtonProps } from '@audius/harmony'
 
-import { SocialButton, SocialButtonProps } from '../SocialButton'
-
-import styles from './TikTokButton.module.css'
-
-export type TikTokButtonProps = SocialButtonProps
+export type TikTokButtonProps = Partial<SocialButtonProps>
 
 export const TikTokButton = (props: TikTokButtonProps) => {
-  return (
-    <SocialButton
-      leftIcon={<IconTikTok color='staticWhite' />}
-      {...props}
-      className={cn(styles.button, props.className)}
-    />
-  )
+  return <SocialButton socialType='tiktok' {...props} />
```

Comment:
This almost feels like we don't need a separate file for

Reply:
yeah we truly don't, but it's referenced in like 5 spots so it was a bit much. now that things are operational though, ill take another look